static/js/lib/learning_resources.js

```diff
@@ -26,15 +29,24 @@ export const availabilityLabel = (availability: ?string) => {
 }
 
 export const resourceLabel = (resource: string) => {
-  return concat(capitalize(resource), "s")
+  switch (resource) {
+  case LR_TYPE_USERLIST:
+    return "Learning Paths"
+  default:
+    return concat(capitalize(resource), "s")
+  }
 }
 
 export const maxPrice = (course: Course | Bootcamp) => {
   const price = Math.max(...course.prices.map(price => price.price))
   return price > 0 ? `$${price}` : "Free"
 }
 
-export const minPrice = (course: LearningResource | Course | Bootcamp) => {
-  const price = Math.min(...course.prices.map(price => price.price))
+export const minPrice = (
+  course: LearningResourceSummary | Course | Bootcamp | Program | UserList
+) => {
+  // $FlowFixMe
```

Comment:
instead of the `$FlowFixMe` here (we should try to use these _somewhat_ sparingly) you could type the `course` as `Object`

Reply:
👍